gensim/models/sldamodel.py

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2011 Radim Rehurek <radimrehurek@seznam.cz>
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+
+import logging
+import numpy as np
+import six
+from scipy.special import gammaln, psi  # gamma function utils
+from scipy.special import polygamma
+from six.moves import xrange
+
+from gensim import interfaces, utils, matutils
+from gensim.matutils import dirichlet_expectation
+from gensim.models import basemodel, CoherenceModel
+
+# log(sum(exp(x))) that tries to avoid overflow
+try:
```

Comment:
Useless try/catch, for current minimal required SciPy version `from scipy.misc import logsumexp` is enough.